packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -63,10 +69,40 @@ class OpPerfTelemetry {
         this.deltaManager.on("disconnect", () => {
             this.opSendTimeForLatencyStatisticsForMsnStatistics = undefined;
             this.clientSequenceNumberForLatencyStatistics = undefined;
+            this.clientSequenceNumberForSittingInProcessingQueue = undefined;
             this.connectionOpSeqNumber = undefined;
             this.firstConnection = false;
         });
 
+        this.deltaManager.outbound.on("push", (messages) => {
+            for (const msg of messages) {
+                if (msg.type === MessageType.Operation && msg.clientSequenceNumber % numOpsToThrottle === 1) {
+                        this.clientSequenceNumberForSittingInProcessingQueue = msg.clientSequenceNumber;
+                        this.opTimeSittingInInboundQueue = Date.now();
+                }
+            }
+        });
+
+        this.deltaManager.inbound.on("push", (message: ISequencedDocumentMessage) => {
+            if (message.type === MessageType.Operation) {
```

Comment:
There is no need to check for MessageType.Operation here. It's redundant and may create bugs in the future if condition above change.